README.rst

```diff
@@ -29,8 +29,18 @@ Features
 
 Installation
 -------------
-See doc/install.txt in the source code archive.
-Python 2.7.2 or later is needed.
+
+You likely want to use 
+
+``pip install -e git+git@github.com:linkcheck/linkchecker.git#egg=linkchecker``
+
+or 
+
+``pip install --user -e git+git@github.com:linkcheck/linkchecker.git#egg=linkchecker``
+
+See doc/install.txt in the source code archive for partially outdated but more detailed instructions.
+
+Python 2.7.2 or later is needed. It doesn't work with Python 3 yet, see `#40 <https://github.com/linkcheck/linkchecker/pull/40>`_ for details.
```

Comment:
that's good, but you should keep your commits seperated topically

also clarify the "why" or at least the "what" of the commit in the commitlog instead of just "fix #88" - the commitlog should stand on its own without the issue queue...

Reply:
I've update my commit message accordingly. I'd argue that updating installation instructions is a topic well sized for one commit.

Reply:
sure, but i don't see what updating installation instructions have to do with python 3 support...